server/app/views/applicant/ProgramIndexTemplate.html

```diff
@@ -33,51 +33,23 @@
   ></head>
   <body>
     <div th:replace="~{applicant/NavigationFragment :: pageHeader}"></div>
-    <main>
+    <main class="grid-container padding-0">
```

Comment:
**note:** The `grid-container` class adds some padding but our alerts don't have that padding so this aligns the banner with the alerts which show up right above it.